tests/integration/study_data_blueprint/test_renewable.py

```diff
@@ -211,6 +229,33 @@ def test_lifecycle(
         assert res.status_code == 200, res.json()
         assert res.json() == fr_solar_pv_cfg
 
+        # =============================
+        #  RENEWABLE CLUSTER DUPLICATION
+        # =============================
+
+        new_name = "Duplicate of SolarPV"
+        res = client.post(
+            f"/v1/studies/{study_id}/areas/{area_id}/renewable/{fr_solar_pv_id}?new_cluster_name={new_name}",
+            headers={"Authorization": f"Bearer {user_access_token}"},
+        )
+        # asserts the config is the same
+        assert res.status_code in {200, 201}
+        duplicated_config = copy.deepcopy(fr_solar_pv_cfg)
```

Comment:
Since the dictionary is "flat", `copy.deepcopy` is useless.
You can use:

```python
duplicated_config = {**fr_solar_pv_cfg}
```

or

```python
duplicated_config = fr_solar_pv_cfg.copy()
```